packages/react-server/src/ReactFlightServer.js

```diff
@@ -4982,11 +4994,11 @@ function forwardDebugInfo(
             // $FlowFixMe[cannot-write]
             debugAsyncInfo.stack = debugStack;
           }
-          request.pendingDebugChunks++;
+          request.pendingChunks++;
           emitDebugChunk(request, id, debugAsyncInfo);
         }
       } else {
-        request.pendingDebugChunks++;
+        request.pendingChunks++;
```

Comment:
Can't we just move this into `emitDebugChunk`? It's very confusing to have it outside, whereas `request.pendingDebugChunks++` is inside.